parameters/production-parameters.json

```diff
@@ -0,0 +1,100 @@
+{
+  "orgName": "BountyDAO",
+  "tokenName": "BountyDAO Token",
+  "tokenSymbol": "BDT",
+  "ContributionReward": [{}],
+  "SchemeRegistrar": [{}],
+  "StandAloneContracts": [
+    {
+      "name": "PolkaCurve",
+      "fromArc": true,
+      "arcVersion": "0.0.1-rc.39"
+    },
+    {
+      "name": "ContributionRewardExt",
```

Comment:
this is standalone contract or custom ?

Reply:
There is an issue that both ContributionRewardExt and Competition are interdependent for their initialisation. To solve this, I deploy it in the standalone contracts but initialise in the custom schemes

Reply:
ok